Legacy/BootPlatform/DxeIpl/X64/Canary.nasm

```diff
@@ -0,0 +1,98 @@
+; @file
+; Copyright (C) 2021, ISP RAS. All rights reserved.
+;
+; This program and the accompanying materials
+; are licensed and made available under the terms and conditions of the BSD License
+; which accompanies this distribution.  The full text of the license may be found at
+; http://opensource.org/licenses/bsd-license.php
+;
+; THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+; WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+;
+BITS 64
+
+#include <AutoGen.h>
+extern ASM_PFX(_ModuleEntryPointReal)
+
+section .data
+align 8
+global ASM_PFX(__security_cookie)
+ASM_PFX(__security_cookie):
+  dq 0
+
+global ASM_PFX(__stack_chk_guard)
+ASM_PFX(__stack_chk_guard):
+  dq 0
```

Comment:
I have just realised that we have two distinct variables for stack canaries, when we can have just one. Could you please rewrite this by  dropping the extra dq and leaving just one assignment? This way both symbols will be aliases to the same memory.